cmk/gui/plugins/wato/check_parameters/interfaces.py

```diff
@@ -114,6 +155,10 @@ def _vs_single_discovery():
                                 title=_("Generate service labels for discovered interfaces"),
                             ),
                         ),
+                        (
+                            "labels_conditions",
+                            _vs_labels_conditions(),
+                        )
                     ],
                     optional_keys=["labels"],
```

Comment:
```suggestion
                    optional_keys=["labels", "labels_conditions"],
```

Hi @lbuhleie, thanks for your PR.

After trying it out with our internal test suite, I noticed the configuration didn't pass. 

From what I can tell, it is just missing label_conditions as an optional key because otherwise it is expected to be there with the default parameters for the valuespec.

I'd give it a local test to be sure it doesn't affect anything (from what I can tell, it shouldn't).